app/views/home/feed.xml.builder

```diff
@@ -0,0 +1,19 @@
+atom_feed do |feed|
+  feed.title 'Awesome Foundation – Winning Projects'
+  feed.updated @projects[0].updated_at unless @projects.empty?
+
+  @projects.each do |project|
+    feed.entry(project) do |entry|
+      entry.title "#{project.chapter.name} – #{project.title}"
+      entry.content(project.about_project, type: 'html')
+
+      entry.author do |author|
+        author.name project.name
+      end
+
+      if project.has_images?
+        photo.link(href: project.primary_image.url, rel: 'enclosure', type: 'image/jpeg')
```

Comment:
This breaks because `photo` isn't actually defined yet. You probably want `project.primary_image` here.

You also don't technically need the conditional either - if you don't include it we'll just use the default [placeholder image](http://d13mwkvpspjvzo.cloudfront.net/assets/no-image-main.png) if the project doesn't have any images.